src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.1.0
+-----
+
+ * Added `Symfony\Component\HttpFoundation\InputBag`
+ * Deprecated retrieving non-string values using `InputBag::get()` ( use `InputBag::all()`, or `InputBag::getRaw()` instead )
```

Comment:
` * Deprecated retrieving non-string values using `InputBag::get()`, use `InputBag::all()` if you need access to collection of values`
(rebase needed)